src/services/explorer.ts

```diff
@@ -13,6 +13,8 @@ export interface Explorer {
     chartsType: string
   ) => void;
   refreshWalletData: () => void;
+  hasInfinityLoader: () => boolean;
```

Comment:
```suggestion
  hasInfiniteLoader: () => boolean;
```

It's an infinite loader (endless), not a loader that loads an infinity)
Infinite is an adjective, infinity is a noun :D